src/frontend/apps/desk/src/features/mail-domains/components/forms/CreateMailboxForm.tsx

```diff
@@ -157,6 +234,7 @@ const Form = ({
             render={({ fieldState }) => (
               <Input
                 aria-invalid={!!fieldState.error}
+                aria-required="true"
```

Comment:
You can just used the input properties, you don't need all this check I think.


```suggestion
                aria-required="true"
                required
```